### PR TITLE
bugfix: fix workaround for wrong spans in extension method call

### PR DESCRIPTION
**Pull request by kasiaMarek**
resolves: https://github.com/scalameta/metals/issues/5921

### CODE REVIEW

File: mtags/src/main/scala-3/scala/meta/internal/pc/MetalsInteractive.scala [hunk 1]
```diff
@@ -345,6 +345,26 @@ object MetalsInteractive:
     }
   end ApplySelect
 
+  object ExtensionMethodCallSymbol:
+    def unapply(tree: Tree)(using Context): Option[(Ident | Select)] =
```
Comment (jkciesluk) [inline mtags/src/main/scala-3/scala/meta/internal/pc/MetalsInteractive.scala:349]:
```suggestion
    def unapply(tree: Tree)(using Context): Option[Symbol] =
```
Could this return a symbol directly?

**Review by jkciesluk — APPROVED** [review]
LGTM

File: mtags/src/main/scala-3/scala/meta/internal/pc/MetalsInteractive.scala (hunk 1 above)
Comment (kasiaMarek) [reply mtags/src/main/scala-3/scala/meta/internal/pc/MetalsInteractive.scala:349]: I use the tree in `PcCollector`.

Comment (jkciesluk) [reply mtags/src/main/scala-3/scala/meta/internal/pc/MetalsInteractive.scala:349]: Ah right, thanks, I missed it.

**Pull request merged**